src/routes/bulk/post.rs

```diff
@@ -36,48 +36,38 @@ const EMAIL_TASK_BATCH_SIZE: usize = 1;
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
 struct TaskInput {
-	job_id: i32,
-	input: CheckEmailInput,
-}
-
-/// Endpoint request body.
-#[derive(Clone, Debug, Deserialize, Serialize)]
-struct CreateBulkRequestBody {
+	// fields for CheckEmailInput
+	input: Vec<String>,   // chunk of email from request
```

Comment:
```suggestion
	to_emails: Vec<String>,   // chunk of email from request. This always has at most `EMAIL_TASK_BATCH_SIZE` items.
```

Too many fields called "inputs".